pkgs/applications/networking/mailreaders/evolution/evolution-ews/default.nix

```
@@ -0,0 +1,47 @@
+{ stdenv, fetchurl, gnome3, cmake, gettext, intltool, pkg-config, evolution-data-server
+, sqlite, gtk3, webkitgtk, libgdata }:
+
+stdenv.mkDerivation rec {
+  pname = "evolution-ews";
+  version = "3.36.4";
+
+  src = fetchurl {
+    url = "mirror://gnome/sources/evolution-ews/${stdenv.lib.versions.majorMinor version}/${pname}-${version}.tar.xz";
+    sha256 = "0zfq02h3r1qbxak04i49564q4s2ykvkgcyc3krjgndan9lq3kvvn";
+  };
+
+  PKG_CONFIG_EVOLUTION_SHELL_3_0_ERRORDIR = "${placeholder "out"}/share/evolution/errors";
+  PKG_CONFIG_EVOLUTION_SHELL_3_0_PRIVLIBDIR = "${placeholder "out"}/lib/evolution";
+  PKG_CONFIG_CAMEL_1_2_CAMEL_PROVIDERDIR = "${placeholder "out"}/lib/evolution-data-server/camel-providers";
+  PKG_CONFIG_LIBEDATA_BOOK_1_2_BACKENDDIR = "${placeholder "out"}/lib/evolution-data-server/addressbook-backends";
+  PKG_CONFIG_LIBEDATA_CAL_2_0_BACKENDDIR = "${placeholder "out"}/lib/evolution-data-server/calendar-backends";
+  PKG_CONFIG_LIBEBACKEND_1_2_MODULEDIR = "${placeholder "out"}/lib/evolution-data-server/registry-modules";
+  PKG_CONFIG_EVOLUTION_SHELL_3_0_MODULEDIR = "${placeholder "out"}/lib/evolution/modules";
+  PKG_CONFIG_EVOLUTION_DATA_SERVER_1_2_PRIVDATADIR = "${placeholder "out"}/share/evolution-data-server";
+
+  nativeBuildInputs = [ cmake gettext intltool pkg-config ];
+
+  buildInputs = [
+    evolution-data-server gnome3.evolution
+    sqlite libgdata
+    gtk3 webkitgtk
+  ];
+
+  cmakeFlags = [
+    "-DWITH_MSPACK=OFF"
+    "-DCMAKE_BUILD_TYPE=Debug"
+  ];
```

It would be nicer to use https://github.com/NixOS/nixpkgs/blob/7b2bdbea75a202c416941798aeb9544e2596fa3d/pkgs/development/tools/build-managers/cmake/setup-hook.sh#L92:

```suggestion
  ];

  cmakeBuildType = "Debug";
```

But perhaps we should use the default release builds.